src/ch/imedias/rsccfx/view/RsccRequestView.java

```diff
@@ -86,6 +86,8 @@ private void initFieldData() {
     keyGeneratorPane.setText("Key generator");
     keyGeneratorPane.setExpanded(true);
 
+    generatedKeyFld.setPromptText("Generating key...");
```

Comment:
I thought of changing this to "Working..." but maybe it would make more sense to make the textfield larger and center the text? Maybe I'll have a look into it with @l-marchesi in the project week